scholia/app/templates/taxon_tree.sparql

```diff
@@ -0,0 +1,52 @@
+#defaultView:Graph
+SELECT
+  ?child ?childLabel
+  ?rgb 
+  ?parent ?parentLabel
+WITH {
+  SELECT ?child ?rgb ?parent WHERE {
+    { 
+      # Parent taxons
+      SELECT ?child ?rgb ?parent  WHERE {
+        SERVICE gas:service {
+          gas:program gas:gasClass "com.bigdata.rdf.graph.analytics.BFS" ;
+                  gas:in wd:Q15978631 ;
```

Comment:
@egonw I think the home sapiens issue is just here.

Reply:
Actually, in two more lines. See the extra patch.